include/picolibrary/testing/unit/gpio.h

```diff
@@ -246,6 +249,17 @@ class Mock_Internally_Pulled_Up_Input_Pin {
             return m_mock_internally_pulled_up_input_pin->enable_pull_up();
         }
 
+        /**
+         * \brief Disable the pin's internal pull-up resistor.
+         *
+         * \return Nothing if enabling the pin's internal pull-up resistor succeeded.
+         * \return An error code if enabling the pin's internal pull-up resistor failed.
```

Comment:
`\return` documentation references enabling the pin's internal pull-up resistor when it should be referencing disabling it.